test/EndToEndTests/Common/Microsoft.OData.Client.E2E.TestCommon/Common/TestHttpClientRequestMessage.cs

```diff
@@ -1,5 +1,5 @@
 ﻿//-----------------------------------------------------------------------------
-// <copyright file="HttpClientRequestMessage.cs" company=".NET Foundation">
+// <copyright file="TestHttpClientRequestMessage.cs" company=".NET Foundation">

```

Comment:
what's the reason to change the class name? I am just curious about the root cause?